services/blockchain-indexer/shared/dataService/business/schemas.js

```diff
@@ -13,24 +13,46 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+const { getRegisteredModules, MODULE } = require('../../constants');
 const { requestConnector } = require('../../utils/request');
 
 let allSchemas;
 
+const getAuthMultiSigRegMsgSchema = async () => {
+	const registeredModules = await getRegisteredModules();
+	const isAuthModuleRegistered = registeredModules.includes(MODULE.AUTH);
+
+	if (isAuthModuleRegistered) {
+		const authMultiSigRegMsgSchema = await requestConnector('getAuthMultiSigRegMsgSchema');
+		return {
+			moduleCommand: `${MODULE.AUTH}:registerMultisignature`,
+			param: 'signatures',
+			...authMultiSigRegMsgSchema, // schema
```

Comment:
The actual schema should be identified by the `schema` property within a messageSchema.
```suggestion
			schema: authMultiSigRegMsgSchema,
```

Reply:
SDK returns following structure:

```
{
    schema: {
        ...
    }
}
```

Reply:
Please rename the variable/refactor the code to make it more intuitive. `authMultiSigRegMsgSchema` is misleading.

Reply:
Refactored.